### PR TITLE
MDK tutorials - updated with latest changes

### PR DESCRIPTION
**Pull request by jitendrakansal**
MDK tutorials - updated with latest changes

### CODE REVIEW

File: tutorials/cp-mobile-dev-kit-add-validation/cp-mobile-dev-kit-add-validation.md [hunk 1]
```diff
@@ -144,11 +144,28 @@ Deploy the updated application to your MDK client.
...
+
+1. Either click the highlighted button or refresh the web page to load the changes.
+
+    !![MDK](img_5.3.png)
+
+    >If you see the error `404 Not Found: Requested route ('xxxxxtrial-dev-nsdemosampleapp-approuter.cfapps.xxxx.hana.ondemand.com') does not exist.` while accessing the web application, make sure that in your space cockpit, highlight applications are in started state.
+
+    >!![MDK](img_5.6.png)
+
+2. Update a record with no contain of **@** in Email property throws a validation failure message.
```
Comment (MichaelCzcz) [inline tutorials/cp-mobile-dev-kit-add-validation/cp-mobile-dev-kit-add-validation.md:157]: Can you please rephrase to make it better understandable? Grammar seems to be a bit off too.

File: tutorials/cp-mobile-dev-kit-annotations/cp-mobile-dev-kit-annotations.md [hunk 2]
```diff
@@ -116,7 +116,7 @@ This step includes creating the mobile development kit project in the editor.
 
     !![MDK](img_3.5.png)
 
-    Regardless of whether you are creating an online or offline application, this step is needed app to connect to an OData service. When building an Mobile Development Kit application, it assumes the OData service created and the destination that points to this service is setup in Mobile Services and SAP Cloud Platform.
+    Regardless of whether you are creating an online or offline application, this step is needed app to connect to an OData service. When building an Mobile Development Kit application, it assumes the OData service created and the destination that points to this service is setup in Mobile Services and SAP Business Technology Platform.
```
Comment (MichaelCzcz) [inline tutorials/cp-mobile-dev-kit-annotations/cp-mobile-dev-kit-annotations.md:119]: Are there some words missing around "app"?

File: tutorials/cp-mobile-dev-kit-offline-app/cp-mobile-dev-kit-offline-app.md [hunk 3]
```diff
@@ -86,51 +85,63 @@ This is how the project structure looks like within the workspace.
...
+>!![MDK](img_2.1.png)
 
 [DONE]
 [ACCORDION-END]
 
-[ACCORDION-BEGIN [Step 3: ](Deploy and activate the application)]
+[ACCORDION-BEGIN [Step 3: ](Deploy the application)]
 
-So far, you have learned how to build an MDK application in the SAP Business Application Studio editor. Now, we deploy this application definition to Mobile Services.
+So far, you have learned how to build an MDK application in the SAP Business Application Studio editor. Now, we deploy this application definition to Mobile Services and Cloud Foundry to consume it as Mobile and Web application respectively.
```
Comment (MichaelCzcz) [inline tutorials/cp-mobile-dev-kit-offline-app/cp-mobile-dev-kit-offline-app.md:111]: Please say: " Now, you will deploy....".

File: tutorials/cp-mobile-dev-kit-annotations/cp-mobile-dev-kit-annotations.md [hunk 4]
```diff
@@ -175,7 +175,7 @@ This step includes creating the mobile development kit project in the editor.
 [ACCORDION-END]
 
 
-[ACCORDION-BEGIN [Step 5: ](Deploy and activate the application)]
+[ACCORDION-BEGIN [Step 5: ](Deploy the application)]
 
 So far, you have learned how to build an MDK application in the SAP Business Application Studio editor. Now, we deploy this application definition to Mobile Services.
```
Comment (MichaelCzcz) [inline tutorials/cp-mobile-dev-kit-annotations/cp-mobile-dev-kit-annotations.md:180]: Don't say "we", say "you" instead

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

**Comment by jitendrakansal** [comment]
@MichaelCzcz thanks for reviewing, updated with proposed changes.

**Comment by MichaelCzcz** [comment]
@jitendrakansal You have missed one requested change.

**Comment by jitendrakansal** [comment]
@MichaelCzcz Please check now
missed  "Now, we deploy" part

**Pull request merged**